components/plugins/heapsize.rs

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use syntax::ext::base::{Annotatable, ExtCtxt};
```

Comment:
I think this file should have a top-level comment briefly explaining what it's for, and when to use the `ignore_heap_size` annotation.